aep/general/0144/aep.md.j2

```diff
@@ -1,24 +1,36 @@
 # Array fields
 
-Representing lists of data in an API is trickier than it often appears. Users
-often need to modify lists in place, and longer data series within a single
-resource pose a challenge for pagination.
+Within API, methods that operate on schemas containing arrays require
+consideration on the strategies for updating elements within the list.
+
+For larger arrays of resources, consider [pagination][/pagination].
 
 ## Guidance
 
 Resources **may** use array fields where appropriate.
 
-```typescript
-interface Book {
-  // The resource name for the book.
-  name: string;
+{% tab proto %}
+
+```proto
+message Book {
+  option (google.api.resource) = {
+    type: "library.googleapis.com/Book"
+    pattern: "publishers/{publisher}/books/{book}"
+  };
 
-  // The authors of the book.
-  authors: string[];
+  string name = 1 [(google.api.field_behavior) = IDENTIFIER];
+
+  repeated string authors = 2;
 }
 ```
 
-- Array fields **must** use a plural field name.
+{% tab oas %}
+
+**Note:** OAS guidance not yet written
+
+{% endtabs %}
+
+- Array fields **should** use a plural field name.
```

Comment:
Why is this weakened from must to should?  Because it can't reliably be enforced by a linter?

Reply:
the main reason is that we wouldn't break a client by violating this.

Like all shoulds - this isn't bad guidance, but we have to be careful about using *must*